### PR TITLE
Do not specify strict versions

### PR DESCRIPTION
**Pull request by Mimetis**
### Summary :memo:
The last version of `fastapi` is not working with `aad_fastpi` as we are specifying strictly the version

image

Replacing in requirements.txt all specific version (`==`) with at least version (`>=`)

### CODE REVIEW

**Comment by Mimetis** [comment]
Hey @dorlugasigal, I'm trying to fix the version issue, but without success.
I'm not able to run the tests. Still wondering why, if you have any idea ?

File: aad_fastapi/aad_options.py [hunk 1]
```diff
@@ -27,18 +27,18 @@ class AzureAdSettings(BaseSettings):
...
     """
 
-    client_id: str = Field(None, description="Client id", env="CLIENT_ID")
-    client_secret: str = Field(None, description="Client Secret", env="CLIENT_SECRET")
-    authority: str = Field(None, description="login authority", env="AUTHORITY")
-    domain: str = Field(None, description="Domain name", env="DOMAIN")
-    tenant_id: str = Field(None, description="Tenant Id", env="TENANT_ID")
-    scopes: str = Field(None, description="Scopes", env="SCOPES")
-    vault_name: str = Field(None, description="Global Vault Url", env="VAULT_NAME")
+    client_id: str = Field("", description="Client id", env="CLIENT_ID")
```
Comment (Mimetis) [inline aad_fastapi/aad_options.py]: Hey @dorlugasigal, I'm not sure about that, replacing `None` with `""`, but at least it's working
Maybe `Optional` would be better (but It did not work for me, I guess I made a mistake somewhere)

Comment (Mimetis) [reply aad_fastapi/aad_options.py]: Using `Optional[str]` resolves this.

File: tests/conftest.py [hunk 2]
```diff
@@ -14,12 +14,14 @@
 from aad_fastapi.roles.role_requirement import RoleRequirement
 
 os.environ["CLIENT_ID"] = "01010101-aaaa-bbbb-acdf-020202020202"
+os.environ["CLIENT_SECRET"] = "a_secret_from_azure_ad"
```
Comment (dorlugasigal) [inline tests/conftest.py]: is this used?

**Review by dorlugasigal — APPROVED** [review]
glad you fixed that, i wanted to find time to do that as well :)

File: tests/conftest.py (hunk 2 above)
Comment (Mimetis) [reply tests/conftest.py]: oh, no, was just a test on optional things

**Pull request merged**